### PR TITLE
Replace scrapy with beautiful soup

### PR DESCRIPTION
**Pull request by LuptonM**
https://uktrade.atlassian.net/jira/software/projects/KLS/boards/359?assignee=63343adf88ed2ebef97c64ac&selectedIssue=KLS-827

### CODE REVIEW

File: requirements_browser.txt [hunk 1]
```diff
@@ -105,10 +88,7 @@ exceptiongroup==1.1.2
     # via
     #   trio
     #   trio-websocket
-    # via
-    #   great-magna-tests-shared
-    #   tldextract
-great-magna-tests-shared @ file:///app/great_magna_tests_shared
+great-magna-tests-shared @ file:///Users/marclupton/Desktop/DIT/great-magna-tests/great_magna_tests_shared
```
Comment (hal274) [inline requirements_browser.txt:91]: Why does this have your local file path? Will this work on another machine?

File: requirements_functional.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #
 #    pip-compile requirements_functional.in
 #
--e ./great_magna_tests_shared/
+-e file:///Users/marclupton/Desktop/DIT/great-magna-tests/great_magna_tests_shared
```
Comment (hal274) [inline requirements_functional.txt:7]: same here

File: requirements_browser.txt (hunk 1 above)
Comment (LuptonM) [reply requirements_browser.txt]: something changed with the make file i think as i dont think the make commands were working locally before. have now changed the paths back

**Review by bsabhilash — APPROVED** [review]
LGTM!

**Pull request merged**